### ISSUE TITLE
Access to test iOS Accessibility

### ISSUE THREAD

**Issue by bischoffa** [department-of-veterans-affairs/va-mobile-app#9100]
## Context:
I have been unable to access TestFlight since May.

I can access App Store Connect without issue (and can even send myself a new invitation email to TestFlight), but when I attempt to access anything relating to TestFlight (via the email invitation or a specific build), I get the following error message (also attached a screenshot):

**Couldn't redeem TestFlight invitation code "[code here]"**
The invitation has been revoked or is invalid. Request a new invitation code from the developer.

<img width="369" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBelRrQlE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--6b1129db0e5dd2daf331ec79817b3d01a07e6b72/IMG_0011.PNG" alt="Screenshot of an error message in TestFlight" />

I also get a similar error message if I tap the "View in TestFlight" button in the invitation email if I have already previously installed TestFlight on the device. It says:

**Couldn't Load App**
The invitation has been revoked or is invalid. Request a new invitation from the developer.

I have tried various combinations of: restarting the device, deleting/re-installing TestFlight, copying the redemption code in, pasting the redemption code in, tapping the button in the email after installing TestFlight, etc. As Tim mentioned, I have even been removed as a user and then invited again; but no matter what we have tried, I cannot see any builds in TestFlight. TestFlight asks me to enter a redemption code to access the builds, but anytime I enter the code, I get the error in the screenshot above.

**Comment by brea11y** [comment]
**Update:** I was able to regain access to TestFlight using my personal Apple ID on Friday (7/19), but I am still troubleshooting the TestFlight access with my Ad Hoc Apple ID.
